js/src/lib/interfaces/Types.ts

```diff
@@ -431,16 +439,135 @@ export const PIPELINE_DATA = ensureRecordOfPipelines({
 		color:    "green",
 	},
 	"reinforcement-learning": {
-		name:     "Reinforcement Learning",
-		modality: "rl",
-		color:    "red",
+		name:           "Reinforcement Learning",
+		modality:       "rl",
+		color:          "red",
 		hideInDatasets: true,
 	},
-	"structured-data-classification": {
-		name:     "Structured Data Classification",
-		modality: "other",
-		color:    "blue",
+	"tabular-classification": {
+		name:     "Tabular Classification",
+		modality: "nlp",
```

Comment:
is this supposed to be NLP? it's weird no? shouldn't it be `other` or even better a new `structured` modality?

Reply:
Having a `structured` modality sounds good to me

Reply:
Yes, I agree and actually had a comment for exactly this in the `datasets` PR. Will update accordingly

Reply:
Although should modality be `tabular` for alignment with AutoTrain I think

Reply:
`tabular` if we don't include time-series inside of them, `structured` if we do?